mitosheet/src/components/endo/celleditor/CellEditor.tsx

```diff
@@ -493,7 +498,7 @@ const CellEditor = (props: {
                 In the dropdown box, we either show an error, a loading message, suggestions
                 or the documentation for the last function, depending on the cases below
             */}
-            <div className='cell-editor-dropdown-box' style={{width: props.editorState.editorLocation === 'cell' ? `${CELL_EDITOR_WIDTH}px` : '300px'}}>
+            <div className='cell-editor-dropdown-box' style={{width: `${cellEditorWidth}px`}}>
```

Comment:
I don't think we want this in the case of the formula bar because it covers up so much of the data and column headers. Can we just leave this as the regular width in the formula bar?


<img width="1430" alt="Screenshot 2023-04-11 at 12 18 39 PM" src="https://user-images.githubusercontent.com/18709905/231225917-efb0f2ba-8e54-4379-a435-5b57adc8219d.png">